Dockerfile.moc-xdmod

```diff
@@ -97,6 +98,22 @@ RUN chmod 774 /usr/bin/xdmod-openstack-hypervisor \
     && find /etc/httpd -type d -exec chmod 764 {} + \
     && find /etc/httpd -type f -exec chmod 664 {} +
 
+# openshift client
+RUN wget https://downloads-openshift-console.apps.ocp-staging.massopen.cloud/amd64/linux/oc.tar --no-check-certificate \
```

Comment:
Why are you downloading it from the staging cluster? I'm hesitant to introduce a dependency to a staging cluster in the build process of the container.

Reply:
Fair - is there a better place to download it from? The openshift documentation didn't have an ideal public repository, but I could be missing something!

Reply:
I'm getting it from here https://github.com/CCI-MOC/openshift-acct-mgt/blob/fa396d7752ecf21248071f022ae5e43c1ae24176/.github/workflows/functional-tests.yaml#L35-L36

Reply:
Ah, thanks - updated!